### DISCUSSION TITLE
Libre 3 Patch installation schlägt fehl

### DISCUSSION THREAD

**Discussion by tim1401** [maheini/FreeStyle-Libre-3-patch#40] [General]
Hallo,
hat jemand eine Lösung für das problem , wenn man die Gepatchte Libre 3 Version auf einem Samsumg A13 4G 128GB, Androidversion 13, installieren will und die 

Fehlermeldung: Die App wurde nicht installiert, da sie nicht mit dem Spmartphone kompatibel ist.

Es wurde die Installation von unbekannte Apps erlaubt, die Berechtigungen gesetzt, Juggluco und Libre 2 hat Funktioniert.
Libre 3 original wurde vorher deinstalliert, auch da gab es keinen Erfolg.
Was kann ich nocht tun, damit ich die gepatchte Version aufs Handy bekomme?
Danke für eure Hilfe

**Comment by bboi61** [comment]
Wozu noch die gepatchte Libre3 App nutzen, wenn es auch ohne funktioniert? Juggluco kann den Libre 3 Sensor jetzt auch starten, Voraussetzung, ein in der App eingetragenes LibreView Konto.

**Reply by tim1401** [reply]
ok, das Wusste ich nicht, danke für die Info, werde es beim nächsten Sensor 
ausprobieren....
